services-directions-models/src/main/java/com/mapbox/api/directions/v5/models/DirectionsRoute.java

```diff
@@ -185,6 +196,16 @@ public static DirectionsRoute fromJson(String json) {
      */
     public abstract Builder duration(@NonNull Double duration);
 
+    /**
+     * The estimated travel time from origin to destination including delay (road repair,
+     * traffic jam, etc.).
```

Comment:
```suggestion
     * The typical travel time from this route's origin to destination. There's a delay along this route if you subtract this durationTypical() value from the route's duration() value and the resulting difference is greater than 0. The delay is because of any number of real-world situations (road repair, traffic jam, etc).
```